shareprogress/services.py

```diff
@@ -0,0 +1,35 @@
+# Managing Buttons
+def create_button(page_url, button_template, page_title=None, *args):
+    # page_url: The URL of the page to be shared.
```

Comment:
I'd check out this link here for convention: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings